Testing Beta URL by adding environment var in tox (#55)
Helioviewer-Project/python-api: docs/dev_guide.rst

``pytest``
==========

> .. warning::
>     Before running the tests we need to set environment variables ``HELIOVIEWER_API_URL`` to ``https://api.beta.helioviewer.org/v2/``.

Reviewer: ```suggestion
```

> .. warning::
>     Before running the tests we need to set environment variables ``HELIOVIEWER_API_URL`` to ``https://api.beta.helioviewer.org/v2/``.

Reviewer: ```suggestion
.. note::

    We run the test suite with the beta version of the Helioviewer API.
    So if you run **pytest** without it, the tests will fail.
    
    There are two ways to change this. 
    1. Set the environment variable ``HELIOVIEWER_API_URL`` to ``https://api.beta.helioviewer.org/v2/``.
    2. Add ``HELIOVIEWER_API_URL="https://api.beta.helioviewer.org/v2/"`` to the command line when you run **pytest**.
```

Author: I will add it in the next commit.

>     Merging of the PR #55, it is expected that the ``pytest`` is going to fail. This is beacuse the ``pytest`` functions is testing the beta version of the API.

Reviewer: This isn't true.

> .. warning::
>     Merging of the PR #55, it is expected that the ``pytest`` is going to fail. This is beacuse the ``pytest`` functions is testing the beta version of the API.

Reviewer: ```suggestion
```
This isn't true. You can just pass the beta url with pytest.

Author: What do we need to update in dev_guide?

Reviewer: You need to say that they need to set the environment variable to be the beta URL as they run pytest.
